generator/maxspeed_builder.cpp

```diff
@@ -0,0 +1,12 @@
+#include "generator/maxspeed_builder.hpp"
+
+#include "routing/maxspeed_serialization.hpp"
+
+namespace generator
+{
+bool BuildMaxspeed(std::string const & dataPath, std::string const & osmToFeaturePath,
+                   std::string const & maxspeedFilename)
+{
+  return true;
```

Comment:
The right way is to return false for now :)

Reply:
Right! :)